Mathlib/Analysis/PSeries.lean

```diff
@@ -471,4 +471,14 @@ lemma Real.summable_one_div_int_add_rpow (a : ℝ) (s : ℝ) :
   simp_rw [summable_int_iff_summable_nat_and_neg, ← abs_neg (↑(-_ : ℤ) + a), neg_add,
     Int.cast_neg, neg_neg, Int.cast_natCast, summable_one_div_nat_add_rpow, and_self]
 
+theorem summable_pow_div_add {α : Type*} (x : α) [RCLike α] (p q k : ℕ) (hq : 1 < q) :
+    Summable fun n : ℕ => ‖(x ^ p / (↑n + k) ^ q)‖ := by
+  simp_rw [div_eq_mul_inv, norm_mul]
+  apply Summable.mul_left
+  simp_rw [inv_eq_one_div]
```

Comment:
Are we missing `Summable.const_div`?

Reply:
yeah sure, I've added this and some similar results.